src/main/scala/nl.knaw.dans.easy.solr4files/EasyUpdateSolr4filesIndexService.scala

```diff
@@ -33,7 +33,8 @@ class EasyUpdateSolr4filesIndexService(val serverPort: Int, app: EasyUpdateSolr4
         override def probeForCycleClass(classLoader: ClassLoader): (String, LifeCycle) = {
           ("Solr4files-lifecycle", new LifeCycle {
             override def init(context: ServletContext): Unit = {
-              context.mount(new EasyUpdateSolr4filesIndexServlet(app), "/fileindex")
+              context.mount(new UpdateServlet(app), "/fileindex")
+              context.mount(new SearchServlet(app), "/filesearch")
```

Comment:
My plan for the search API was 'easy.dans.knaw.nl/fileitem/search', so have a 'fileitem' resource with a search on it. 
Maybe we have Jan and Linda give their opinions, but it seems easy to change at this stage in the development. 
As soon as we have the API in production, we are kind of stuck with it.

Reply:
The proxies should map `http://deasy.dans.knaw.nl/files/search` to `http://deasy.dans.knaw.nl:20150/filesearch` (and `http://deasy.dans.knaw.nl/files/download` to another servlet)